build/pipelines/fuzz.yml

```diff
@@ -0,0 +1,61 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    exclude:
+      - docs/*
+      - samples/*
+      - tools/*
+
+pr: none
+
+#     0.0.yyMM.dd##
+#     0.0.1904.0900
+name: 0.0.$(Date:yyMM).$(Date:dd)$(Rev:rr)
+
+stages:
+  - stage: Build_Fuzz_Config
+    displayName: Build Fuzzers
+    dependsOn: []
+    condition: succeeded()
+    jobs:
+    - template: ./templates/build-console-fuzzing.yml
+      parameters:
+        platform: x64

```

Comment:
For my own edification - I get that this calls the build in `build-console-fuzzing`. I don't get how this tells that build to only build the `Fuzzing` config (and therefore only build conhost)

Reply:
At the top of `build-console-fuzzing.yml`, there's some predefined parameters:
```yaml
parameters:
  configuration: 'Fuzzing'
  platform: ''
  additionalBuildArguments: ''
```
I think you'd be able to override the `configuration` parameter from here, but like, that would be a bad thing to do haha. The pipeline was designed to run in the fuzzing config so something would 100% fail.

Reply:
NIT: in that case, it probably should not be configurable 😉